tests/foreman/api/test_subscription.py

```diff
@@ -234,6 +234,7 @@ def test_positive_delete_manifest_as_another_user(function_org, default_sat):
 
 
 @pytest.mark.tier2
+@pytest.mark.rhel_ver_match('.*compose')
 def test_positive_subscription_status_disabled(
```

Comment:
Is this intntional? that you wanted to run this test only for `composes` ?

Reply:
sorry, this was already a part of my testing. not a part of this PR. removed